File: iter2/src/Page/SyllabusPage.java
package Page;

import System.FunctionType;
import System.SystemMessage;

public class SyllabusPage extends Page{
	
	public SyllabusPage(String content) {
		super(content);
		setType(PageType.SYLLABUS_PAGE);
		setName("Syllabus Page Student");
	}

	@Override
	public SystemMessage runPage() {
		showContent();
		takeInput();
		return new SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE, null);
	}
}

File: python_development/Page/SyllabusPage.py
from python_development.Page import Page
from python_development.Page import PageType
from python_development.System import FunctionType
from python_development.System import SystemMessage


class SyllabusPage(Page):

    def __init__(self, content):
        super().__init__(content)
        self.setType(PageType.SYLLABUS_PAGE)
        self.setName("Syllabus Page Student")


    def runPage(self):
        self.showContent()
        self.takeInput()
        return SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE, None)
